arelle/plugin/validate/ESEF/common.py

```diff
@@ -0,0 +1,5 @@
+"""Shared functions and type hints."""
+from __future__ import annotations
+from collections.abc import Callable
+
+TypeGetText = Callable[[str], str]
```

Comment:
We can probably move this even higher up in the hierarchy but I've placed it here for now.